hooks/terraform_docs.sh

```diff
@@ -42,8 +42,8 @@ function replace_old_markers {
 
   # Determine the appropriate sed command based on the operating system (GNU sed or BSD sed)
   sed --version &> /dev/null && SED_CMD=(sed -i) || SED_CMD=(sed -i '')
-  "${SED_CMD[@]}" -e "s/^${old_insertion_marker_begin}$/${insertion_marker_begin}/" "$file"
-  "${SED_CMD[@]}" -e "s/^${old_insertion_marker_end}$/${insertion_marker_end}/" "$file"
+  "${SED_CMD[@]}" -e "s/^${old_insertion_marker_begin}$/${insertion_marker_begin//\//\\/}/" "$file"
+  "${SED_CMD[@]}" -e "s/^${old_insertion_marker_end}$/${insertion_marker_begin//\//\\/}/" "$file"
```

Comment:
Did you mean just add `//\//\\/` as for L45?

```suggestion
  "${SED_CMD[@]}" -e "s/^${old_insertion_marker_end}$/${insertion_marker_end//\//\\/}/" "$file"
```

Reply:
Yeah, that is a copy fail.